internal/cli/ops_manager_clusters_list.go

```diff
@@ -37,30 +36,22 @@ func (opts *opsManagerClustersListOpts) init() error {
 }
 
 func (opts *opsManagerClustersListOpts) Run() error {
-	result, err := opts.cloudManagerClustersListRun()
+	if opts.ProjectID() == "" {
+		result, err := opts.store.ListAllProjectClusters()
+		if err != nil {
+			return err
+		}
+		return json.PrettyPrint(result)
+	}
 
+	clusterConfigs, err := opts.store.GetAutomationConfig(opts.ProjectID())
```

Comment:
This is a small drive by fix/clean up, we weren't checking for the error of the get and that was giving me then a nil pointer exception